### ISSUE TITLE
docker-compose instructions do not work

### ISSUE THREAD

**Issue by fedorov** [webanno/webanno#1361]
**Describe the bug**
While following the instructions for using WebAnno with `docker-compose`, as discussed here: https://webanno.github.io/webanno/releases/3.5.7/docs/admin-guide.html#_connecting_to_a_mysql_database, there are numerous runtime errors in the log, and trying to connect to localhost:8080 results in "no data received".

**To Reproduce**
1. Go to https://webanno.github.io/webanno/releases/3.5.7/docs/admin-guide.html#_connecting_to_a_mysql_database
2. Follow the steps starting with "We provide a docker-compose example file, which combines the two containers."

**Expected behavior**
Opening localhost:8080 connects to the working instance of WebAnno.

**Screenshots**
N/A

**Please complete the following information:**
 - Today's version of documentation
 - OS: macOS
 - Browser: chrome

**Additional context**
It appears that `docker-compose.yml` is using `webanno/webanno-snapshots:latest`, which is 2 years old: https://github.com/webanno/webanno/blob/master/webanno-webapp/src/main/docker/docker-compose.yml#L31. 

Replacing that line with `webanno/webanno:3.5.6` results in everything working as expected. Can someone confirm this is the right way to fix this issue? I am a bit surprised no one ran into this, since it would seem that `docker-compose` is absolutely the easiest way to configure an instance.

Overall, thanks for the great documentation - I was able to get started in rather short time!

**Comment by reckart** [comment]
WebAnno 3.5.7 is the latest release. Adjusting the compose script to use a current release instead of a SNAPSHOT build is definitely a good idea. And as you have noticed, the SNAPSHOTs are not really updated (anymore).

I guess the proper way to go would be to embed the compose script into the asciidoc documentation and to auto-generate the right version string into it.

**Comment by fedorov** [comment]
Great, thanks for confirming! 

 am not familiar with the details of asciidoc, but let me know if you want me to make a PR to update the docker image location to `webanno/webanno:latest`.

**Comment by reckart** [comment]
Right, that would also be an option :) I'm not too familiar with docker. If we change the image location to `webanno/webanno:latest`, does that mean that Docker will always use the latest version when the container is restarted? I mean, does it mean we essentially would auto-update and take the control when to switch to the latest version away from the user?

**Comment by fedorov** [comment]
I believe it does mean the docker will always use the latest version, but it is also more error prone, since it is impossible to control the environment - the version can effectively switch underneath. From what I know, in production setting, the use of `latest` is discouraged. When I experimented with it, I was also not sure if `latest` is the same as the version with the highest number, and was also not sure about your version numbering conventions (e.g., some projects use odd numbers to designate pre-releases), so that's why I tried `3.5.6`.

I think it is fair to change to `latest`, and add a note in the documentation that production deployments should use the latest numbered version instead.

But I also do not consider myself too familiar with the docker, and would happily defer to an expert on the subject matter.
